packages/kbn-unified-data-table/src/components/data_table_column_header.tsx

```diff
@@ -23,7 +23,11 @@ interface DataTableColumnHeaderProps {
 
 export const DataTableColumnHeader: React.FC<DataTableColumnHeaderProps> = (props) => {
   const { columnDisplayName } = props;
-  const columnToken = getRenderedToken(props);
+  const columnToken = useMemo(
+    () => getRenderedToken(props),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
To avoid needing to override the lint rule, could we instead destructure the required props in the line above and pass them in directly?